packages/neutrino-preset-jest/src/index.js

```diff
@@ -6,13 +6,25 @@ const { tmpdir } = require('os');
 const clone = require('lodash.clonedeep');
 const loaderMerge = require('neutrino-middleware-loader-merge');
 
+function getFinalPath(path) {
+  if (path[0] === '/') {
+    return path;
+  }
+  if (path[0] === '.') {
+    return join('<rootDir>', path);
+  }
+  return join('<rootDir>', 'node_modules', path);
+}
+
 function normalizeJestOptions(jestOptions, config, args) {
   const options = clone(jestOptions);
   const aliases = config.resolve.alias.entries() || {};
-
   Object
     .keys(aliases)
-    .map(key => Object.assign(options.moduleNameMapper, { [key]: join('<rootDir>', aliases[key]) }));
+    .map((key) => {
```

Comment:
Since final path is only used one place here, we could re-write this as:

```js
.map(key => Object.assign(options.moduleNameMapper, {
  [`${key}(.*)`]: `${getFinalPath(aliases[key])}$1`
}));
```